### PR TITLE
Rename charm and adapt description

### PR DESCRIPTION
**Pull request by arturo-seijas**

### CODE REVIEW

File: metadata.yaml [hunk 1]
```diff
@@ -1,14 +1,11 @@
-name: discourse-k8s
+name: discourse
```
Comment (mthaddon) [inline metadata.yaml:1]: We can't rename the charm, as there are existing deployments using this name, and also there's a machine charm with the same name (however poorly maintained).

File: metadata.yaml [hunk 2]
```diff
@@ -1,14 +1,11 @@
...
-  Use it as a mailing list, discussion forum, long-form chat room and more!
-  https://www.discourse.org
+summary: Discourse is the discussion platform built for the next decade of the Internet.
+description: Mailing list, discussion forum, chat room and more!
 docs: https://discourse.charmhub.io/t/discourse-documentation-overview/3773
 maintainers:
-  - Jay Kuri <jay.kuri@canonical.com>
-  - Gareth Woolridge <gareth.woolridge@canonical.com>
+  - launchpad.net/~canonical-is-devops
+issues: https://github.com/canonical/discourse-operator/issues
```
Comment (mthaddon) [inline metadata.yaml:8]: The project link is https://github.com/canonical/discourse-k8s-operator/issues

**Review by mthaddon — CHANGES_REQUESTED** [review]

File: metadata.yaml (hunk 1 above)
Comment (arturo-seijas) [reply metadata.yaml]: Noted

**Review by gregory-schiano — APPROVED** [review]
LGTM

**Review by merkata — APPROVED** [review]

**Review by mthaddon — APPROVED** [review]
LGTM

**Pull request merged**